vcd/resource_vcd_org_test.go

```diff
@@ -77,9 +80,20 @@ func testAccCheckOrgDestroy(s *terraform.State) error {
 			continue
 		}
 
-		org, err := govcd.GetOrgByName(conn.VCDClient, rs.Primary.Attributes["name"])
-		if org != (govcd.Org{}) || err != nil {
-			return fmt.Errorf("org with name %s was found", rs.Primary.Attributes["name"])
+		deleted := false
+		var org govcd.Org
+		var err error
+		for N := 0; N < 10; N++ {
+			org, err = govcd.GetOrgByName(conn.VCDClient, rs.Primary.Attributes["name"])
+			deleted = (org == (govcd.Org{})) && (err != nil)
+			if deleted {
+				break
+			}
+			time.Sleep(time.Second)
```

Comment:
Would really like to see consistent handling of timeouts in the adapter. Would you think about it and file a separate issue for this? It should lay out the idea how can we resolve this, like what configurable parameter we need and which places in the code need to be refactored.

Reply:
I am working on it. But we need some discussion on this matter.